### DISCUSSION TITLE
Tempat Diskusi Ramadhan Code Fest IMPHNEN 2024!

### DISCUSSION THREAD

**Discussion by ilhamjaya08** [IMPHNEN/Ramadhan-Code-Fest#2] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ilhamjaya08** [comment]
cek ombak

**Comment by ZTRdiamond** [comment]
malas ngoding

**Comment by ilhamjaya08** [comment]
yayayay
